### ISSUE TITLE
Closing bracket is missing at authenticationendpoint/basicauth.jsp file

### ISSUE THREAD

**Issue by deshankoswatte** [wso2/product-is#12900]
**Describe the issue:**
There is a syntax issue in the source code at the <IS_HOME>/repository/deployment/server/webapps/authenticationendpoint/basicauth.jsp file. Starting at line number 262, in the following snippet of code (The second div tag is missing the bracket ">"):
```
<div class="field">
   <div class="g-recaptcha"
        data-sitekey="<%=Encode.forHtmlContent(reCaptchaKey)%>" // Closing bracket not here.
   </div>
</div>
```

**How to reproduce:**
Cannot reproduce an error (even if you enable re captcha). 

**Expected behavior:**
The syntax should be properly fixed as follows:
```
<div class="field">
   <div class="g-recaptcha"
        data-sitekey="<%=Encode.forHtmlContent(reCaptchaKey)%>"> // Closing bracket added here.
   </div>
</div>
```

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
 - Product Version: IS-5.10.0
 - OS: Ubuntu 20.04.3 LTS 
 - Database: H2
 - Userstore: LDAP

---

**Comment by nilasini** [comment]
@deshankoswatte can we close this issue? Isn't it reproducible in latest product?

**Comment by deshankoswatte** [comment]
@nilasini yes, we can close the issue since I have already fixed in the 5.10.0 branch. Furthermore, as you said this issue is not reproducible in the mast latest product.
